web/frontend/components/PublishButton.vue

```diff
@@ -1,45 +1,110 @@
-              <span v-else class="fa-li"><font-awesome-icon icon="fa-solid fa-xmark fa-fw" /></span>
-              Cover Photo
-            </li>
-            <li>
-              <span v-if="this.publishCheck.descriptionExists" class="fa-li"><font-awesome-icon icon="fa-solid fa-check fa-fw" /></span>
-              <span v-else class="fa-li"><font-awesome-icon icon="fa-solid fa-xmark fa-fw" /></span>
-              Description 
-            </li>
-          </ul>
-          We also encourage you to share your work with your colleagues or on
-          social media! Your book is an important contribution to open education
-          for law and we hope you'll help others find it and use it in their own
-          classrooms. Because you are a verified professor, your book will
-          automatically be surfaced on H2O's search page and will appear in web
-          searches.
+        <aside class="publish-modal">
+        <div v-if="!publishSuccess">
+          <div v-if="publishCheck.isVerifiedProfessor" id="prof-prompt">
+            <p>
+              You're almost ready to publish your book! The following elements
+              are optional, but can make your book more discoverable by students
+              and colleagues:
+            </p>
+            <ul class="fa-ul">
+              <li v-if="this.publishCheck.coverImageFlag">
+                <span v-if="this.publishCheck.coverImageExists" class="fa-li"
+                  ><font-awesome-icon icon="fa-solid fa-check fa-fw"
+                /></span>
+                <span v-else class="fa-li"
+                  ><font-awesome-icon icon="fa-solid fa-xmark fa-fw"
+                /></span>
+                Cover Photo
+              </li>
+              <li>
+                <span v-if="this.publishCheck.descriptionExists" class="fa-li"
+                  ><font-awesome-icon icon="fa-solid fa-check fa-fw"
+                /></span>
+                <span v-else class="fa-li"
+                  ><font-awesome-icon icon="fa-solid fa-xmark fa-fw"
+                /></span>
+                Description
+              </li>
+            </ul>
+          </div>
+          <div v-else>
+            <p>Are you ready to publish your book?</p>
+            <p>
+              Once published, your casebook will be available to anyone with the
+              link.
+            </p>
+          </div>
         </div>
+        <div v-else-if="this.publishCheck.isVerifiedProfessor">
 
-        <div class="modal-footer">
-          <button class="modal-button cancel" @click="cancelPublish">Go Back</button>
-          <button class="modal-button confirm" @click="confirmPublish">Publish</button>
+          <p>
+            Use this link to share your book with students and colleagues:
+            {{ this.canonicalUrl }}
+          </p>
+          <p>
+            Because you are a verified professor, your book will be
+            surfaced in H2O's search page and will appear in web searches. We
```

Comment:
Tiny suggestion: maybe this should link to the search page? Also I can't remember if there is a delay in indexing; if yes, could be worth saying that search results aren't available immediately?

Reply:
We run `fab refresh_search_index` every five minutes fwiw